src/CallableHandler.php

```diff
@@ -122,4 +75,13 @@ public static function execute($callable, array $arguments = [])
             throw $exception;
         }
     }
+
+    protected static function getResolver()
```

Comment:
Oh, I think I get it. Well, I wouldn't use this CallableHandler in the middlewares. By default I would instantiate a ReflectionResolver in the constructor. If a container is set as a resolver, I overwrite the default instance and wrap it in a ContainerResolver instance. That's my idea so far.

An alternative would be the syntax that we discussed earlier (pass the container as a third parameter to the CallableHandler).

I am happy to implement whichever you prefer.